### ISSUE TITLE
Clean up the sandbox

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo-loihi#18]
Once the current round of refactoring is done, we should go through the `sandbox` and either promote files to tests and/or examples, or remove files that are no longer relevant.

**Comment by drasmuss** [comment]
Sandbox was removed in https://github.com/nengo/nengo-loihi/commit/7f92b5ba5bbe911436796c6ebd5a123da49db5dd
